index.js

```diff
@@ -18,6 +18,8 @@ mongoose.Promise = global.Promise;
 mongoose.connect(process.env.MONGODB_URI || ('mongodb+srv://' + username + ':' + password + '@techradarhtw-tdfaj.mongodb.net/test'));
 
 app.use(bodyParser.json());
+app.use(cors());
```

Comment:
difficult to explain ... but before ( without this cors) Leonie has the problem like that she cannot access our DB from localhost 3000 to 5000... we got an 404 error and something like :not allowed to access the resource . this import fixed it.